docs/_docs/configuration.md

```diff
@@ -602,12 +602,13 @@ file or on the command-line.
 
 ```yaml
 # Where things are
-source:       .
-destination:  ./_site
-plugins_dir:  _plugins
-layouts_dir:  _layouts
-data_dir:     _data
-includes_dir: _includes
+source:          .
+destination:     ./_site
+collections_dir: .
```

Comment:
this should be `""` instead of a `.`

Reply:
@ashmaroli Isn't it the current directory? (ie: same as source?)

Reply:
I was pointing at a discrepancy in `lib/jekyll/configuration.rb` above and the documentation here..

In `lib/jekyll/configuration.rb` above, its `{"source" => Dir.pwd, "collections_dir" => ""}` but here its `source:  .` and `collections_dir:  .`